### ISSUE TITLE
Add Code of Conduct link to event page

### ISSUE THREAD

**Issue by dcwalk** [shaunagm/bridge_troll#21]
Would be nice to have the Code of Conduct referenced in the preset text!

**Comment by shaunagm** [comment]
Good call!  It's in the sign ups, but it should be on the event page as well.  Should be a quick fix, will add this next time I work on the project if no one else has gotten to it by then.
